utils/warnings.ml

```diff
@@ -220,8 +220,14 @@ let is_active x = (!current).active.(number x);;
 let is_error x = (!current).error.(number x);;
 
 let parse_opt error active flags s =
-  let set i = flags.(i) <- true in
-  let clear i = flags.(i) <- false in
+  let set i =
+    let i = if i = 25 then 8 else i in
+    flags.(i) <- true
+  in
+  let clear i =
+    let i = if i = 25 then 8 else i in
+    flags.(i) <- false
```

Comment:
So people who used to disable explicitly warning 25 would now disable warning 8.  This is quite bad, no?

I think I would prefer fixing the man page, and perhaps issue a deprecation warning if people try to touch warning 25, although this might be really overkill.